### PR TITLE
new oidc provider api

### PR DESCRIPTION
**Pull request by irisked**
New oidc providers api for tenants.

### CODE REVIEW

File: source/includes/_tenants.md [hunk 1]
```diff
@@ -2091,3 +2091,303 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+> Example Request:
+
+```shell
+ curl {{ api_base_url }}/v2/tenants/{TENANT_ID}/oidc-providers \
+   -X POST \
+   -u {EMAIL}:{APIKEY} \
+   -H 'Content-Type: application/json' -d '
+   {  
+     "data":{  
+       "type":"oidc-provider,
```
Comment (HannaTrotsenko) [inline source/includes/_tenants.md]: missing "

File: source/includes/_tenants.md [hunk 2]
```diff
@@ -2091,3 +2091,303 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+       "attributes":{
+         "issuer": "{ISSUER}"
```
Comment (HannaTrotsenko) [inline source/includes/_tenants.md]: missing ,

File: source/includes/_tenants.md [hunk 3]
```diff
@@ -2091,3 +2091,303 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+         "client_id":"{CLIENT_ID}",
+         "client_secret":"{CLIENT_SECRET}",
```
Comment (HannaTrotsenko) [inline source/includes/_tenants.md]: unneeded ,

File: source/includes/_tenants.md [hunk 4]
```diff
@@ -2091,3 +2091,303 @@ This request is authorized for the users with the `tenants.oauth_clients.delete`
...
+> Example Response:
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/json
+{  
+  "data":{
+    "id": "5c80e6b9bb0d200011333d92",
+    "type": "oidc-provider",
+    "attributes": {
```
Comment (HannaTrotsenko) [inline source/includes/_tenants.md]: "attributes": {
            "issuer": "https://accounts.google.com",
            "tenant_id": "56c207adb9121181e650c0ef",
            "client_id": "387712951585-e9cf9nbqs1pptj1bblp2tgbtl845rhfd.apps.googleusercontent.com",
            "client_secret": "CFeRH9LEe9y5umnIFxrQxqJK",
            "response_type": "code",
            "config": {
                "create_new_user_if_not_found": true
            }
        }

File: source/includes/_tenants.md (hunk 3 above)
Comment (irisked) [reply source/includes/_tenants.md]: fix

File: source/includes/_tenants.md (hunk 4 above)
Comment (irisked) [reply source/includes/_tenants.md]: fix

File: source/includes/_tenants.md (hunk 2 above)
Comment (irisked) [reply source/includes/_tenants.md]: fix

**Pull request merged**